community/neo4j/src/test/java/migration/StartOldDbOn3_4AndCreateFusionIndexIT.java

```diff
@@ -155,6 +239,17 @@ private void verifyIndexes( GraphDatabaseAPI db, Label label ) throws Exception
         assertEquals( 34, countIndexedNodes( db, label, KEY1, KEY2 ) );
     }
 
+    private void verifyAfterAdditionalUpdate( GraphDatabaseAPI db, Label label ) throws Exception
+    {
+        // There should be an index for the label and KEY1 containing 100 nodes
```

Comment:
Comment is wrong, I realize that it's twice those numbers now that one more round of updates have been added. Comment could reflect that, or be removed.